### PR TITLE
Add 2.12 roadmap

### PR DESCRIPTION
**Pull request by sivel**
##### SUMMARY
Add 2.12 roadmap

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
docs/docsite/rst/roadmap/ROADMAP_2_12.rst

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by webknjaz — APPROVED** [review]

File: docs/docsite/rst/roadmap/ROADMAP_2_12.rst
Comment (ssbarnea) [inline docs/docsite/rst/roadmap/ROADMAP_2_12.rst:30]: only 5 weeks between first pre-release and the release? IMHO, that is too little. I would personally like to see prerelease on pypi appearing at least 2 months before the release.

If there is a feature freeze, why there is no package published for it?

Based on my experience very few people are testing pre-releases and even fewer are testing code from devel branch.

For several Ansible related projects I do have CI/CD pipelines which test prereleases of dependencies, which allow us to find breakages. Still for this to function we need to make the pre-release publishing something common, I would argue that as common as patch-releases on stable branches.

File: docs/docsite/rst/roadmap/ROADMAP_2_12.rst
Comment (ssbarnea) [inline docs/docsite/rst/roadmap/ROADMAP_2_12.rst:42]: I would have prefered to hear that ansible-test is slowly moving outside core, so it can be used as a standalone development tool, with its own release cycle.

**Review by samccann — COMMENTED** [review]
If you are changing the filename, you'll also need to change it here:
https://github.com/ansible/ansible/blob/devel/docs/docsite/sphinx_conf/ansible_conf.py#L101

**Comment by sivel** [comment]
> If you are changing the filename, you'll also need to change it here:
> https://github.com/ansible/ansible/blob/devel/docs/docsite/sphinx_conf/ansible_conf.py#L101

Done.

**Review by samccann — APPROVED** [review]
LGTM. Are we waiting on any other reviews or can this be merged?

**Comment by sivel** [comment]
> Are we waiting on any other reviews or can this be merged?

I'm waiting on more reviews.  I'll merge when ready.

**Review by thaumos — APPROVED** [review]

File: docs/docsite/rst/core_index.rst
Comment (webknjaz) [inline docs/docsite/rst/core_index.rst:90]: FTR the `.rst` suffix is unnecessary in doctrees. It's better to drop it in the future.

**Pull request merged**
